### PR TITLE
Added members from the SciTools org that work for the Met Office.

### PR DESCRIPTION
**Pull request by pelson**
This is the canonical place that the CLA checker looks.

https://github.com/scitools-incubator/scitools-cla-checker

### CODE REVIEW

**Comment by pelson** [comment]
I thought I'd be a little bit ironic @bjlittle, and make a commit in your name on this PR (I also have a commit in the PR, but I'm on the list). As you can see, you need to merge this PR in order for the status to go green 😱  (infinite loop)

File: contributors.json [hunk 1]
```diff
@@ -1,6 +1,20 @@
 {
 "contributors": [
 { "profile_name":"ajdawson", "organisation":""},
+{ "profile_name":"rhattersley", "organisation":"Met Office"},
+{ "profile_name":"pelson", "organisation":"Met Office"},
```
Comment (ajdawson) [inline contributors.json]: @pelson is now listed twice, is that the desired state?

Comment (pelson) [inline contributors.json]: It is a reasonable state - they are CLAs for two distinct organisations.
INAL but: I think that covers section 4 of the CLA:


> 4. You represent that you are legally entitled to grant the above licence. If your employer(s) has rights to intellectual property that you create that includes your Contributions, you represent that you have received permission to make Contributions on behalf of that employer, that your employer has waived such rights for your Contributions to the Met Office.

Comment (pelson) [inline contributors.json]: I guess technically, the CLA is between the individual and SciTools, so perhaps the organisation is redundant in the JSON. In that situation, I'd be tempted to have it removed altogether from ``contributors.json``.

**Pull request merged**